llvm/projects/ton-compiler/cpp-sdk/tvm/slice.hpp

```diff
@@ -28,7 +28,7 @@ class __attribute((tvm_tuple)) slice {
     return __builtin_tvm_sdeq(sl_, v);
   }
   operator __tvm_slice() const { return get(); }
-private:
```

Comment:
Would it be better to provide getter to this field instead of explicit access?

Reply:
It would be better, but after this patch, class `Slice` began to convert into a tuple via `to_std_tuple`. This conversion employs [structured binding](https://github.com/NilFoundation/zkllvm-circifier/blob/456d87ee5f994a5196c4bc1e72aefa127adb92e9/llvm/projects/ton-compiler/cpp-sdk/tvm/to_std_tuple.hpp#L25) technique, which is available only for public class variables.
It is possible to make private data available for structured binding, but it requires some additional magic, that I believe doesn't worth it. It’s also worth noting that public variables is a commonplace for TON SDK, since a converting struct to a tuple is a core technique there.

Reply:
Ok. Let's stay as is